sdk/metrics-testing/src/main/java/io/opentelemetry/sdk/testing/assertj/metrics/ExponentialHistogramAssert.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.assertj.metrics;
+
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.ExponentialHistogramData;
+import io.opentelemetry.sdk.metrics.data.ExponentialHistogramPointData;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.AbstractIterableAssert;
+import org.assertj.core.api.Assertions;
+
+/** Test assertions for {@link ExponentialHistogramData}. */
+public class ExponentialHistogramAssert
+    extends AbstractAssert<ExponentialHistogramAssert, ExponentialHistogramData> {
+
+  protected ExponentialHistogramAssert(ExponentialHistogramData actual) {
+    super(actual, ExponentialHistogramAssert.class);
+  }
+
+  /** Ensures that {@code aggregation_temporality} field is {@code CUMULATIVE}. */
+  public ExponentialHistogramAssert isCumulative() {
+    isNotNull();
+    if (actual.getAggregationTemporality() != AggregationTemporality.CUMULATIVE) {
+      failWithActualExpectedAndMessage(
+          actual,
+          "aggregationTemporality: CUMULATIVE",
+          "Exepcted Histogram to have cumulative aggregation but found <%s>",
+          AggregationTemporality.CUMULATIVE,
```

Comment:
I believe this is copy-pasted from my own mistaken use of this method.   You should drop this parameter, as you only have one %s which is the actual temporality.

Reply:
Done!